causallearn/utils/cit.py

```diff
@@ -59,9 +59,18 @@ def _unique(column):
         }
 
     def kci(self, X, Y, condition_set):
+        if condition_set == None:
+            condition_set = []
+        if type(X) == int:
+            X = [X]
```

Comment:
Better with an `elif` to ensure X is some iterable.

And then `X = list(X)` - otherwise `self.data[:, X]` does not support `X` as e.g., `set`, or `tuple` with only one element.